### DISCUSSION TITLE
Keyframes without using symbols?

### DISCUSSION THREAD

**Discussion by PlumpyPotato** [keiwando/toonsquid#274] [Q&A] [answered]
Are symbols the only way to put keyframes on **frames with different drawings**? Just curious if maybe there's a better or quicker way to do so~

**Comment by Cat-Butler** [comment, marked as answer]
Well, often it is not needed because the other drawings act as the keyframes - want the Thingy™ in a different place or at a different scale? In the new drawing just draw it in the different place / scale.

However, there will be times when you do want different drawings moved together e.g. draw a walk cycle in place, translate to move, or draw a Thingy™ rolling up to the edge of a surface, then falling off the edge (cliff, table, flat earth - combine with cat to push small objects off).
You can do this in at least two ways:

1. In the same timeline, wrap your drawings layer within [a Transform layer hierarchy](https://toonsquid.com/handbook/transform/transform_layer/), then keyframe the Transform layer - good for something that keeps changing as it moves and/or scales.

2. Convert [your drawings layer into a symbol](https://toonsquid.com/handbook/symbols/create_symbol/#create-in-timeline), delete the original, then insert and keyframe the symbol - this is probably the best choice for cycling animation (cycle in place, create symbol, keyframe the symbol layer).

**Reply by PlumpyPotato** [reply]
Your response not only solved my doubt, but also responded another one that was lurking behind my mind!(about the hierarchy usage)
Thank you so much! \(ᗒᗊᗕ)/
